diff --git a/views/layout/default.blade.php b/views/layout/default.blade.php
index 8097e81..2a2769d 100644
--- a/views/layout/default.blade.php
+++ b/views/layout/default.blade.php
@@ -100,6 +100,9 @@
 		Grocy.Mode = '{{ GROCY_MODE }}';
 		Grocy.BaseUrl = '{{ $U('/') }}';
 		Grocy.CurrentUrlRelative = "/" + window.location.href.split('?')[0].replace(Grocy.BaseUrl, "");
+		var grocyBaseUrl = new URL(Grocy.BaseUrl, window.location.origin);
+		Grocy.BaseUrl = grocyBaseUrl.href;
+		Grocy.CurrentUrlRelative = "/" + window.location.pathname.replace(grocyBaseUrl.pathname, "");
 		Grocy.View = '{{ $viewName }}';
 		Grocy.Currency = '{{ GROCY_CURRENCY }}';
 		Grocy.EnergyUnit = '{{ GROCY_ENERGY_UNIT }}';
